src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -154,15 +153,13 @@ public synchronized BibEntry getEntryByKey(String key) {
         return back;
     }
 
+    /**
+     *
+     * @param key
+     * @return list of entries that contains the given key
```

Comment:
Please rework comment. "Collects entries having the specified BibTeX key and returns these entries as list. The order of the entries is the order they appear in the database"